emuu-client/src/components/UserAuthentication/newRegister.js

```diff
@@ -98,12 +160,16 @@ function Register() {
                     id="password"
                     autoComplete="new-password"
                     value={password}
-                    onChange={(e) => setPassword(e.target.value)}
+                    className="register-input"
+                    placeholder="Password"
+                    onChange={(e) => {setPassword(e.target.value); document.querySelector("#password-err").style.color = "black";document.querySelector("#password-err").innerHTML = "<small> Password must be at least 8 characters with 1 special character and 1 uppercase character </small>" }}
```

Comment:
Why is there a super long line of code? it makes it hard to read what's going on. Next time, format it so new lines of code start on the next line.